sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/Sample4_AnalyzeConversationWithOptions.md

```diff
@@ -31,8 +31,7 @@ ConversationsProject conversationsProject = new ConversationsProject("Menu", "pr
 Response<AnalyzeConversationTaskResult> response = client.AnalyzeConversation(
     textConversationItem,
```

Comment:
Shouldn't this just be a string now?

Reply:
Also, I wonder if the return should just be `Response<AnalyzeConersationResult>`. The "Task" adds nothing.

Reply:
`AnalyzeConversationTaskResult` is used as a return type in the rest client. both are different.